### PR TITLE
Include 'plugin:' prefix in managed item source

### PR DESCRIPTION
**Pull request by aibaars**
@nickfyson This should make the behaviour the same as calling the `getSource(Plugin)` method.

### CODE REVIEW

File: src/main/java/com/semmle/jira/addon/util/JiraUtils.java [hunk 1]
```diff
@@ -193,7 +193,8 @@ public static void createConfigKeyCustomField() throws GenericEntityException {
 
       builder.setManaged(true);
       builder.setConfigurationItemAccessLevel(ConfigurationItemAccessLevel.LOCKED);
-      builder.setSource(Constants.PLUGIN_KEY);
+      builder.setSource(
+          ManagedConfigurationItemService.SOURCE_PREFIX_PLUGIN + Constants.PLUGIN_KEY);
```
Comment (nickfyson) [inline src/main/java/com/semmle/jira/addon/util/JiraUtils.java]: Needs the same prefix added in the check on line 170.

Comment (aibaars) [reply src/main/java/com/semmle/jira/addon/util/JiraUtils.java]: Ah thanks for spotting that!

Comment (aibaars) [reply src/main/java/com/semmle/jira/addon/util/JiraUtils.java]: I amended the commit, it should be better now.

**Comment by nickfyson** [comment]
LGTM

**Pull request merged**